cmd/limactl/shell.go

```diff
@@ -41,6 +41,10 @@ func newShellCommand() *cobra.Command {
 	return shellCmd
 }
 
+// envShell contains the user's shell, as specified by the password database.
+// http://man.openbsd.org/login.1#SHELL
```

Comment:
I linked the first man page I found that was documenting the SHELL env var.
I can also either link to manpages.org, or remove the link altogether.

Reply:
Marking as resolved because OpenBSD's man page for `login(1)` seems to be the only publicly available documentation to have ever documented the `SHELL` environment variable.

If this is causing friction I can also remove the comment.